src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -103,6 +103,17 @@ const mapManifestServiceUrlDeprecation: ConfigDeprecation = (settings, fromPath,
   return settings;
 };
 
+const opsLoggingEventDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
```

Comment:
I took a similar approach as to https://github.com/elastic/kibana/pull/87939#discussion_r561321179 but open to other deprecation implementations.